dlib/CMakeLists.txt

```diff
@@ -353,7 +356,10 @@ if (NOT TARGET dlib)
       endif()
 
       if (DLIB_JPEG_SUPPORT)
-         # try to find libjpeg 
+         # try to find libjpeg
+         if(HUNTER_ENABLED) 
```

Comment:
`if(HUNTER_ENABLED)` is not needed, `hunter_add_package` will check it internally. Take a look carefully at this example: https://docs.hunter.sh/en/latest/overview/compatibility.html